AOBJ/type/zif_aff_aobj_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_aobj_v1
+
+  PUBLIC.
+
+  TYPES: ty_write_class TYPE c LENGTH 40.
+  TYPES: ty_delete_class TYPE c LENGTH 40.
+  TYPES: ty_reload_class TYPE c LENGTH 40.
+
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Name of executable class which writes the data to the archive</p>
+      "! Name of executable class which writes the data to the archive
+      "! $required
+      write_class  TYPE ty_write_class,
+      "! <p class="shorttext">Name of executable class which deletes the data from the tables</p>
+      "! Name of executable class which deletes the data from the tables
+      "! $required
+      delete_class TYPE ty_delete_class,
+      "! <p class="shorttext">Name of executable class which reloads the data to the tables</p>
+      "! Name of executable class which reloads the data to the the tables (may be empty)
+      reload_class TYPE ty_reload_class,
```

Comment:
Is it intended that all the other components are marked as required but not his one?

Reply:
Yes reload classes are not required